src/main/java/de/tum/in/www1/artemis/config/CustomMetricsExtensionConfiguration.java

```diff
@@ -17,6 +18,7 @@
  * CustomMetricsExtensionConfiguration.
  * Configuration for custom Artemis metrics.
  */
+@Profile("core")
```

Comment:
Have you thought about defining "core" in `Constants.java` and reusing the constant and all these different places as we have to use this annotation now in a lot of places and typos are likely to occur.

Reply:
Yes, good idea. WIll do that